declarations_site/chatbot/views.py

```diff
@@ -0,0 +1,119 @@
+import json
+from random import choice
+from django.urls import reverse
+from django.conf import settings
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from spotter.utils import reverse_qs
+from chatbot.utils import ukr_plural, chat_response, simple_search
+
+
+COMMON_ANSWERS = {
+    'дякую': ['Будь ласка.', 'Нема за що!', 'Користуйтесь на здоров\'я', 'Дякую, що користуєтесь.'],
+    'спасибо': ['Пожалуйста', 'Не за что', 'Чому не державною?'],
+}
+
+
+def send_greetings(data):
+    for member in data['membersAdded']:
+        if 'Bot' in member['name']:
+            continue
+        data['from'] = {'id': data['conversation']['id']}
+        message = 'Вітаю, {}!\n\n'.format(member['name'])
+        message += 'Яку декларацію ти шукаєш сьогодні?'
+        chat_response(data, message)
+
+
+def join_res(d, keys, sep=' '):
+    return sep.join([str(d[k]) for k in keys if k in d and d[k]])
+
+
+def search_reply(data):
+    if 'text' not in data or len(data['text']) < 4:
```

Comment:
We have no control over the content of that variable yet happily writing it's content to the DB. At least add a check for the text length